code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -84,10 +84,23 @@
 	max_storage_space = DEFAULT_HUGE_STORAGE * 2
 	matter = list(MATERIAL_STEEL = 10, MATERIAL_GOLD = 10, MATERIAL_DIAMOND = 5, MATERIAL_URANIUM = 5)
 
+/obj/item/weapon/storage/backpack/holding/New()
+	..()
+	item_flags |= BLUESPACE
+
 /obj/item/weapon/storage/backpack/holding/attackby(obj/item/weapon/W as obj, mob/user as mob)
-	if(istype(W, /obj/item/weapon/storage/backpack/holding))
-		to_chat(user, SPAN_WARNING("The Bluespace interfaces of the two devices conflict and malfunction."))
-		qdel(W)
+	if(W.item_flags & BLUESPACE)
```

Comment:
A better way to do all this is to put this code under /obj/item/weapon/storage/ and check if both the item being stored and `src` have the bluespace flag.

Reply:
Indeed, there's awfully many copy and paste in this PR